plugins/apel/src/auditor_apel_plugin/publish.py

```diff
@@ -107,8 +96,8 @@ def run(config, client):
         logging.debug(encoded_sync)
         payload_sync = build_payload(encoded_sync)
         logging.debug(payload_sync)
-        post_sync = send_payload(config, token, payload_sync)
-        logging.debug(post_sync.status_code)
+        # post_sync = send_payload(config, token, payload_sync)
+        # logging.debug(post_sync.status_code)
```

Comment:
Is this intended? For me, it looks like you now have disabled sending the sync message